src/azure-cli/setup.py

```diff
@@ -90,7 +90,7 @@
     'azure-mgmt-iotcentral~=3.0.0',
     'azure-mgmt-iothub~=0.12.0',
     'azure-mgmt-iothubprovisioningservices~=0.2.0',
-    'azure-mgmt-keyvault~=7.0.0b3',
+    'azure-mgmt-keyvault==7.0.0b3',
```

Comment:
this is pined already. btw, why the change? did you met any problem? thanks.

Reply:
@jorgecotillo could you please take look at this problem?

Reply:
Not sure how this got updated, perhaps I pulled the latest and messed up the version? Anyways, this update is no longer part of my PR :)